packages/ui/src/app/pages/WorkingGroups/WorkingGroup/WorkingGroup.stories.tsx

```diff
@@ -96,3 +151,144 @@ export default {
+
+      expect(stakePolicy.toJSON()).toEqual({
+        stakeAmount: 100_0000000000,
+        leavingUnstakingPeriod: 1000,
+      })
+      expect(new BN(rewardPerBlock).toNumber()).toEqual(1000000000)
+
+      expect(openingType).toEqual('Regular')
+      expect(metadataFromBytes(OpeningMetadata, description)).toEqual({ ...WG_OPENING_DATA })
+    })
+  },
+}
+export const CreateOpeningImport: Story = {
+  play: async ({ args, canvasElement, step }) => {
+    const screen = within(canvasElement)
+    const modal = withinModal(canvasElement)
+
+    await userEvent.click(screen.getByText('Add opening'))
+    expect(modal.getByText('Create Opening'))
+    const nextButton = getButtonByText(modal, 'Next step')
+
+    await step('Import File', async () => {
+      const importButton = getButtonByText(modal, 'Import')
+
+      await userEvent.click(importButton)
+
+      const uploadField = await modal.findByText(/Browse for file/)
+      expect(getButtonByText(modal, 'Preview Import')).toBeDisabled()
+
+      await userEvent.upload(
+        uploadField,
+        new File([JSON.stringify(WG_JSON_OPENING)], 'file.json', { type: 'application/json' })
+      )
+    })
+    await step('Check imported data', async () => {
+      expect(await modal.findByText(/File imported successfully, preview your input/))
+
+      const previewImportButton = getButtonByText(modal, 'Preview Import')
+      await waitFor(() => expect(previewImportButton).toBeEnabled())
+      await userEvent.click(previewImportButton)
+
+      expect(await modal.findByLabelText('Opening title'))
+      expect(nextButton).toBeEnabled()
+      await userEvent.click(nextButton)
+
+      expect(await modal.findByText('Opening Duration'))
+      await userEvent.click(modal.getByText('Limited'))
+      expect(nextButton).toBeEnabled()
+      await userEvent.click(nextButton)
+
+      expect(await modal.findByText('Application form'))
+      expect(nextButton).toBeEnabled()
+      await userEvent.click(nextButton)
+
+      expect(await modal.getByLabelText('Staking amount *'))
+      const createButton = getButtonByText(modal, 'Create Opening')
+      await userEvent.click(createButton)
+    })
+    await step('Sign transaction and Create', async () => {
+      expect(await modal.findByText('You intend to create an Opening.'))
+      await userEvent.click(modal.getByText('Sign transaction and Create'))
+    })
+    step('Transaction parameters', () => {
+      const [description, openingType, stakePolicy, rewardPerBlock] = args.onCreateOpening.mock.calls.at(-1)
+
+      expect(stakePolicy.toJSON()).toEqual({
+        stakeAmount: 200_0000000000,
+        leavingUnstakingPeriod: 200,
+      })
+      expect(new BN(rewardPerBlock).toNumber()).toEqual(200000000000)
+
+      expect(openingType).toEqual('Regular')
+      expect(metadataFromBytes(OpeningMetadata, description)).toEqual({
+        ...WG_OPENING_DATA,
+        hiringLimit: 5,
+      })
```

Comment:
Finally here, as a result the same value is expected:
```suggestion
      expect(metadataFromBytes(OpeningMetadata, description)).toEqual(WG_OPENING_METADATA)
```